images/linux/scripts/installers/basic.sh

```diff
@@ -4,90 +4,13 @@
 ##  Desc:  Installs basic command line utilities and dev packages
 ################################################################################
 
+set -e
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
-source $HELPER_SCRIPTS/os.sh
-
-set -e
-
-common_packages="dnsutils
-                 iproute2
-                 iputils-ping
-                 libunwind8
-                 locales
-                 openssh-client
-                 tzdata
-                 upx
-                 zstd
-                 libxkbfile-dev
-                 pkg-config
-                 libsecret-1-dev
-                 libxss1
-                 libgconf-2-4
-                 dbus
-                 xvfb
-                 libgbm-dev
-                 libgtk-3-0
-                 tk
-                 fakeroot
-                 dpkg
-                 rpm
-                 xz-utils
-                 xorriso
-                 zsync
-                 gnupg2
-                 lib32z1
-                 texinfo
-                 libsqlite3-dev
-                 libc++-dev
-                 libc++abi-dev"
-
-cmd_packages="curl
-              file
-              ftp
-              jq
-              netcat
-              ssh
-              parallel
-              rsync
-              shellcheck
-              sudo
-              telnet
-              time
-              unzip
-              zip
-              wget
-              m4
-              bison
-              flex
-              patchelf
-              bzip2
-              sqlite3
-              brotli
-              yamllint"
-
-if isUbuntu20 ; then
-    echo "Install python2"
-    apt-get install -y --no-install-recommends python-is-python2
-fi
-
-echo "Install libcurl"
-if isUbuntu16 || isUbuntu18; then
-   libcurelVer="libcurl3"
-fi
-
-if isUbuntu20 ; then
-    libcurelVer="libcurl4"
-fi
-
-apt-get install -y --no-install-recommends $libcurelVer
-
-# install additional packages only for Ubuntu16.04
-if isUbuntu16; then
-    common_packages="$common_packages
-            libicu55"
-fi
 
+toolsetJson="$INSTALLER_SCRIPT_FOLDER/toolset.json"
```

Comment:
Maybe rename to `toolset`? I think it is not important to specify that it is json

Reply:
@AlenaSviridenko, Should we rename toolsetJson -> toolset in android.sh and azpowershell.sh too?

Reply:
@al-cheb why not? Looks like an easy update

Reply:
Done.